Packs/CVE_2021_44228/Playbooks/playbook-CVE-2021-44228_-_Log4j_RCE.yml

```diff
@@ -2410,12 +2513,12 @@ tasks:
     quietmode: 0
     isoversize: false
     isautoswitchedtoquietmode: false
-  "67":
-    id: "67"
-    taskid: 3ecd0399-c442-4d24-8744-75e4fc0696c1
+  "71":
+    id: "71"
+    taskid: 30d08296-84a7-48fa-8dde-271c04f1f9a2
     type: playbook
     task:
-      id: 3ecd0399-c442-4d24-8744-75e4fc0696c1
+      id: 30d08296-84a7-48fa-8dde-271c04f1f9a2
       version: -1
       name: Cortex XDR - Execute commands
```

Comment:
Need to change the name of the subplaybook task

Reply:
fixed